tests/Integration/Behaviour/Features/Scenario/Order/order_currency.feature

```diff
@@ -0,0 +1,219 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s order --tags multiple-currencies-to-order
+@reset-database-before-feature
+@reboot-kernel-before-feature
+@clear-cache-before-feature
+@multiple-currencies-to-order
+Feature: Multiple currencies for Order in Back Office (BO)
+  In order to manage multiple currencies for orders in BO
+  As a BO user
+  I need to be able to change order informations and have correct results
+
+  Background:
+    Given email sending is disabled
+    Given shop "shop1" with name "test_shop" exists
+    And the current currency is "USD"
+    And country "US" is enabled
+    And country "FR" is enabled
+    And language "French" with locale "fr-FR" exists
+    And I add new currency "currency2" with following properties:
+      | iso_code         | EUR                              |
+      | exchange_rate    | 10.00                            |
+      | name             | My Euros                         |
+      | symbols          | en-US:€;fr-FR:€                  |
+      | patterns         | en-US:¤#,##0.00;fr-FR:#,##0.00 ¤ |
+      | is_enabled       | 1                                |
+      | is_unofficial    | 0                                |
+      | shop_association | shop1                            |
+    And the module "dummy_payment" is installed
+    And I am logged in as "test@prestashop.com" employee
+    And there is customer "testCustomer" with email "pub@prestashop.com"
+    And customer "testCustomer" has address in "US" country
+    And I create an empty cart "dummy_cart" for customer "testCustomer"
+    And I update the cart "dummy_cart" currency to "currency2"
+    And I select "US" address as delivery and invoice address for customer "testCustomer" in cart "dummy_cart"
+    And I add 2 products "Mug The best is yet to come" to the cart "dummy_cart"
+    And I add order "bo_order1" with the following details:
+      | cart                | dummy_cart                 |
+      | message             | test                       |
+      | payment module name | dummy_payment              |
+      | status              | Awaiting bank wire payment |
+    Then order "bo_order1" has 0 payments
+    Then order "bo_order1" should have 2 products in total
+    Then order "bo_order1" should have 0 invoices
+    Then order "bo_order1" should have 0 cart rule
+    Then order "bo_order1" should have following details:
+      | total_products           | 238.00 |
+      | total_products_wt        | 252.28 |
+      | total_discounts_tax_excl | 0.0    |
+      | total_discounts_tax_incl | 0.0    |
+      | total_paid_tax_excl      | 308.00 |
+      | total_paid_tax_incl      | 326.48 |
+      | total_paid               | 326.48 |
+      | total_paid_real          | 0.0    |
+      | total_shipping_tax_excl  | 70.00  |
+      | total_shipping_tax_incl  | 74.20  |
+
+  Scenario: Add cart rule of type 'amount' to an order with secondary currency
```

Comment:
You should add tests on OrderDetail using `@Then product :productName in order :orderReference has following details:` (`checkProductDetailsWithReference`) to make sure the OrderDetail's values are consistent with the Order totals